### PR TITLE
Add fireproofed sites and serp sites to the exclusion list for first party cookies expiry

### PR DESCRIPTION
**Pull request by marcosholgado**
Task/Issue URL: https://app.asana.com/0/0/1203562061689358/f 

### Description
This PR adds fireproofed sites and SERP sites to the exclusion list before changing their expiry date in their 1st party cookies

### Steps to test this PR

- [ ] In `FirstPartyCookiesModifier` line 98, change the value of maxAge to 1 `maxAge = 1`
- [ ] Go to SERP and change the theme
- [ ] Close the app and wait a few seconds seconds before opening the app again
- [ ] Go to SERP, theme should still be the one you set.
- [ ] Doing the same in develop should reset the theme.

### CODE REVIEW

**Comment by marcosholgado** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #2654** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/2654" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/Android/2654?utm_source=stack-comment).

**Review by cmonfortep — APPROVED** [review]
LGTM! also tested weather.com 👍

**Comment by cmonfortep** [comment]
@marcosholgado I've found a weird behavior after testing this. Not sure if related with the PR or a side effect. Let me spend 5 min more minutes before merging it.

**Comment by cmonfortep** [comment]
ok, still unsure why this happens...and not sure if it's related to the hardcoded code used to test this (only happens with SERP, weather.com works fine).

- Go to SERP and change theme (e.g: light -> dark)
- kill the app
- open the app again
- theme is gone ❌ 

BUT, 

- Go to SERP and change theme (e.g: light -> dark)
- make another search
- kill the app
- open the app again
- the theme was persisted 👍

**Comment by marcosholgado** [comment]
Thanks @cmonfortep, ran some tests and wrote https://app.asana.com/0/1202552961248957/1203687282359965/f as a possible follow up.

tl;dr; By leaving SERP right after setting a cookie it doesn't get into the Cookies DB so on restart nothing happens.
